Dfe.Academies.Academisation.Data/AcademisationContext.cs

```diff
@@ -88,15 +90,39 @@ private void SetModifiedAndCreatedDates()
 	{
 		var timestamp = DateTime.UtcNow;
 
-		var entities = ChangeTracker.Entries<BaseEntity>().ToList();
+		var entities2 = ChangeTracker.Entries<AdvisoryBoardDeferredReasonDetails>().ToList();
```

Comment:
Would be good to talk through this to understand it

Reply:
Maybe rename to DeclinedEntities etc

Reply:
renamed with comment to explain why they are there